### DISCUSSION TITLE
Changing transition style while showing screen

### DISCUSSION THREAD

**Discussion by divyansh0201** [SwiftfulThinking/SwiftfulRouting#65] [Q&A]
Hi, 
Is it possible to use any different transition style then the default push transition from trailing edge while using showScreen() method? If yes, how? 
I tried doing - 
        `router.showScreen(.push) { router in
            LoginScreenView(router: router)
                .transition(AnyTransition(.identity)
        }`

but it doesn't change the default transition style! 

Thanks.
@SwiftfulThinking @swiftfulthinking-llc

**Comment by SwiftfulThinking** [comment]
No, the .push is the native segue and is not a Transition.

In version 6.1, there is also "showTransition" which has some additional custom support, such as ".identity"
